pkg/cmd/pr/merge/merge.go

```diff
@@ -322,9 +322,13 @@ func mergeRun(opts *MergeOptions) error {
 
 		var branchToSwitchTo string
 		if currentBranch == pr.HeadRefName {
-			branchToSwitchTo, err = api.RepoDefaultBranch(apiClient, baseRepo)
-			if err != nil {
-				return err
+			if pr.BaseRefName != "" && git.HasLocalBranch(pr.BaseRefName) {
```

Comment:
@vilmibm Do you think we should require that the branch, that we are going to switch to, already exist locally? I can come up with scenarios where users would find either behavior confusing.

Reply:
@samcoe @bstncartwright I would say, let's just switch to the branch without checking for whether it exists locally. If it doesn't exist, git will set up a new local branch that tracks the same-named branch from a git remote. Note that this will fail if multiple git remotes with the same branch name exist, which is likely in case of forks, so it might be even safer to do `git checkout -b <branch> --track <remote>/<branch>` explicitly.

Reply:
@mislav I made the change to use `checkout -b` when the local branch does not exist. Would you mind taking another look? 🙇